packages/fast-usdc-contract/src/exos/settler.ts

```diff
@@ -422,15 +448,37 @@ export const prepareSettler = (
           })();
           if (!dest) return;
 
-          const { reference } = parseAccountId(dest);
+          const { namespace, reference } = parseAccountId(dest);
           const amt = AmountMath.make(USDC, fullValue);
-          const txfrOrSendV =
-            reference === currentChainRef
-              ? E(settlementAccount).send(dest, amt)
-              : E(settlementAccount).transfer(dest, amt, {
-                  forwardOpts: { intermediateRecipient },
-                });
-          void vowTools.watch(txfrOrSendV, this.facets.transferHandler, txHash);
+
+          if (namespace === 'cosmos') {
+            const txfrOrSendV =
```

Comment:
ditto feedback about `tx` having a "k" sound.

consider DRYing this conditional vow